content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,298 @@
+---
+title: 'Get Started: Model registry'
+description:
+  'Set up a Git-based model registry with DVC to track and manage models, their
+  versions and lifecycle stages.'
+---
+
+# Get Started: Model Registry
+
+Just as we use experiment tracking to manage model development, it is a good
+idea to keep a <abbr>model registry</abbr> to manage the lifecycle of the models
+we get from our experiments. Using DVC and [DVC Studio](/doc/studio) we will set
+up a model registry where we can discover, share, deploy and audit all our
+models and which will serve as the single source of truth for our model
+management.
+
+<admon type="tip" id="GTO-tip">
+
+Behind the scenes, DVC Studio uses a command line tool called
+[GTO](<(/doc/gto)>) for most model registry actions.
+
+With GTO you can also set up the model registry locally without DVC Studio. You
+can see how this is done in the expandable "Under the hood" sections in this
+chapter.
+
+</admon>
+
+<details id="follow-along-instructions">
+
+#### 💡 Expand to see how to set things up to follow along with the guide
+
+You can
+[fork our example repository](https://github.com/iterative/example-get-started-experiments/fork)
+and follow the
+[installation steps](https://github.com/iterative/example-get-started-experiments#installation)
+to set it up locally.
+
+To then perform the model registry actions in this guide, follow
+[these steps](/doc/studio/user-guide/experiments/create-a-project#connect-to-a-git-repository-and-add-a-project)
+to sign in to DVC Studio, connect it to your GitHub account and add your forked
+repository as a DVC Studio project.
+
+</details>
+
+## Adding models
+
+Let's now train a model and add it to the model registry. We will be using
+[DVCLive](/doc/dvclive) and add a model using Python code. This will also
+automatically save the model to DVC.
+
+We use the [`log_artifact`](/doc/dvclive/live/log_artifact) method to
+<abbr>cache</abbr> the model with DVC and add it to the the model registry. Open
+the training script `src/train.py` in our example repository and have a look at
+the following code under the `with Live(...)` context:
+
+```python
+with Live(...) as live:
+
+...
+
+    live.log_artifact(
+        path="models/model.pkl",
+        type="model",
+        name="pool-segmentation",
+        desc="This is a Computer Vision (CV) model that's segmenting out swimming pools from satellite images.",
+        labels=["cv", "segmentation", "satellite-images"],
+    )
+```
+
+Here the `path` parameter tells DVC that our model is to be found under
+`"models/model.pkl"`, the `type` parameter is `"model"` and so it will show up
+in DVC Studio (other artifact types will not). The rest of the parameters are
```

Comment:
The Studio team is already working on lifting this restriction. We can keep it for now, but let's at least make an issue to remember to change it.

Reply:
Captured in #4990